dask/array/tests/test_array_core.py

```diff
@@ -2589,6 +2589,28 @@ def test_repeat():
         assert all(concat(d.repeat(r).chunks))
 
 
+@pytest.mark.parametrize('shape, chunks', [
+    ((10,), (1,)),
+    ((10, 11, 13), (4, 5, 3)),
+])
+@pytest.mark.parametrize('reps', [-1, 0, 1, 2, 3, 5, [1, 2]])
+def test_tile(shape, chunks, reps):
+    x = np.random.random(shape)
+    d = da.from_array(x, chunks=chunks)
+
+    try:
+        len(reps)
+    except TypeError:
+        if reps < 0:
+            with pytest.raises(ValueError):
+                da.tile(d, reps)
```

Comment:
Maybe put these checks into a separate test that just runs over errors?

This might keep the main parametrized test simple and easy to understand